README.md

```diff
@@ -284,22 +284,23 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 | [Not a Game Studio](http://www.notagamestudio.com/) | Game development studio focused on the experience of the users. | `Lisboa` |
 
 
-## Industry :factory:
+## Health Care :hospital:
```

Comment:
The idea to make "Medical" category more general to support Nutrium is good but the "Health Care" term does not seem the best option. 

Although nutrition is important when healthcare is provided, nutrition is not generally associated with healthcare. This might be a sector problem but still its like that.

I think a better category would be "Health :heart:" so it's broader than "Medical" without "forcing" a healthcare and nutrition association. What do you think?

Reply:
I agree with the change to "Health ❤️ ". Nonetheless, I've mixed feelings concerning the motivation of changing to "Health", especially due to the following: "nutrition is not generally associated with healthcare.".

“According to WHO, nutrition is a foundation for health and well-being for all, leaving no one behind, and a key element of primary health care, and plays an essential role in prevention.” — https://sdg.iisd.org/news/who-nutrition-is-key-element-of-primary-health-care/

Concerning industries or sectors more specifically, in Crunchbase, for instance, we are able to find "Nutrition" within the "Health Care" category, so the association seems common sense to me.